Tasks/Ovchinnikov/Task3/Task3_10.py

```diff
@@ -0,0 +1,25 @@
+#10. Write a program that takes a list of numbers as input and returns the largest prime number in the list.
+primeNumbersConst = [2, 3, 5, 7, 11, 13, 17, 19, 23, 29, 31, 37, 41, 43, 47, 53, 59, 61, 67, 71, 73, 79, 83, 
+                     89, 97, 101, 103, 107, 109, 113, 127, 131, 137, 139, 149, 151, 157, 163, 167, 173, 179, 
+                     181, 191, 193, 197, 199, 211, 223, 227, 229, 233, 239, 241, 251, 257, 263, 269, 271, 277, 
+                     281, 283, 293]
+
+userDataString = (input("Enter number of elements(separator is space) : "))
+numberList = userDataString.split(' ')
```

Comment:
split without " " parameter will split by space anyway, might as well not pass it